test/test_object.py

```diff
@@ -84,3 +84,31 @@ def test_getitem_s(self):
 
         with pytest.raises(TypeError):
             mod.f([])
+
+    def test_setitem(self):
+        import pytest
+        mod = self.make_module("""
+            HPy_DEF_METH_O(f)
+            static HPy f_impl(HPyContext ctx, HPy self, HPy arg)
+            {
+                HPy key;
+                int result;
+                key = HPyLong_FromLong(ctx, 0);
```

Comment:
same here, better to use a more unique value

Reply:
Being addressed in #27.